### DISCUSSION TITLE
Can't see Vehicles when running generate_traffic.py

### DISCUSSION THREAD

**Discussion by itserror404** [carla-simulator/carla#6625] [Q&A]
Hello,
when running generate_traffic.py, I cannot see any vehicles that are generated. 
I do get the message that some vehicles and people were spawned but I can't see them. Does anyone know how I could go about fixing this please? thanks!

> WARNING: Version mismatch detected: You are trying to connect to a simulator that might be incompatible with this API 
> WARNING: Client API version     = 0.9.14 
> WARNING: Simulator API version  = 0.9.14-6-g43b5e7064 
> ERROR: Spawn failed because of collision at spawn position
> spawned 29 vehicles and 10 walkers, press Ctrl+C to exit.

**Comment by SExpert12** [comment]
You can use spectator like this:

spectator = world.get_spectator()
world_snapshot = world.wait_for_tick() 
spectator.set_transform(vehicle.get_transform())


It will move the focus to the vehicle if they are generated if you are not sure vehicle is generated or not.
